CHANGELOG.md

```diff
@@ -61,6 +61,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
       "otel/sdk/trace".ReadOnlySpan
       "otel/sdk/trace".ReadWriteSpan
 ```
+- Swap "pusher" naming convention to "exporter" for consistent naming across codebase (#1656).
```

Comment:
Can we update this to provide useful info to any user of the project?  Being explicit with the exported things that they will need to pay attention to. E.g.


```suggestion
- `"go.opentelemetry.io/sdk/metric/controller.basic"WithPusher` is replaced with `WithExporter` to provide consistent naming across project. (#1656)
```

Also, this needs to be moved to the `Unreleased` section.

Reply:
definitely, makes sense.